utils.py

```diff
@@ -4,6 +4,9 @@
 import requests, re
 import logging
 from managed_settings import TBPL_TESTS, HOST_ALERT_MANAGER
+import datetime
+
+WEEK   = ['Monday', 'Tuesday', 'Wednesday', 'Thursday',  'Friday', 'Saturday']
```

Comment:
Saturday and Sunday are not really required but since I've added Saturday I'll add Sunday as well to make it look nice